README.md

```diff
@@ -302,6 +302,37 @@ fetcher
 
 For requests from the server, the config object is simply passed into the service being called.
 
+## Using Context Picker
```

Comment:
Sure so this section becomes how do we use context in url and how do we control that using contextPicket.